README.Rmd

```diff
@@ -100,3 +100,10 @@ at:
 ```{r algo-vignette, eval = FALSE}
 vignette("algorithm")
 ```
+
```

Comment:
Maybe put this under a Citing section or so?

Reply:
done